added documentation to clearing dataloader cache (#310)
front-commerce/developers.front-commerce.com: source/docs/magento1/advanced.md

Front-Commerce allows you to send additional headers in all API calls. To do so, you must define the `magento.api.extraHeaders` (for storefront API) and/or `magento.api.extraAdminHeaders` (for admin API) configuration values from a [configuration provider](/docs/advanced/server/configurations.html#What-is-a-configuration-provider).

These additional headers could be useful if you want to add additional context to your queries, depending on the request or to detect Front-Commerce requests from your Magento server.

## Clearing Front-Commerce cache

> To clear the front commerce cache, it suffices to flush cache storage from the admin interface. To do this go to System > Cache Management then click on Flush Cache Storage button on the top right corner. (see below screenshot for more details).

Reviewer: ```suggestion
To clear the Front-Commerce cache, you can flush cache storage from the admin interface. To do this go to System > Cache Management then click on Flush Cache Storage button on the top right corner. (see below screenshot for more details).
```

> <details>

Reviewer: what the point of putting that into a `<details>` tag (details are poorly style
in the doc) and we could perfectly directly show the screenshot

  <summary>Click to reveal screenshot</summary>

>   <p><img src="./clear-fc-cache.jpg" alt="Clear front commerce cache" /></p>

Reviewer: ```suggestion
  <p><img src="./clear-fc-cache.jpg" alt="Clear Front-Commerce cache" /></p>
```
